pkg/delete/delete_loader.go

```diff
@@ -175,6 +175,26 @@ func parseAPIEntry(parsed persistence.DeleteEntry, a api.API) (pointer.DeletePoi
 		log.Warn("Delete entry %q of API type defines config 'id' - only 'name' will be used.")
 	}
 
+	if a.ID == api.KeyUserActionsWeb {
+		if parsed.Scope == "" {
+			return pointer.DeletePointer{}, fmt.Errorf("API of type %q requires a %q, but none was defined", a, "scope")
+		}
+		if v := parsed.CustomValues["actionType"]; v == "" {
+			return pointer.DeletePointer{}, fmt.Errorf("API of type %q requires a %q, but none was defined", a, "actionType")
+		}
+		if v := parsed.CustomValues["domain"]; v == "" {
+			return pointer.DeletePointer{}, fmt.Errorf("API of type %q requires a %q, but none was defined", a, "domain")
```

Comment:
Here we have `%q`, do we also want `%#q`?

Reply:
OR do we want to revert to `%s` for this PR and consider this whole topic in a follow up?

Reply:
for now let's use '%s'